include/mbedtls/psa_util.h

```diff
@@ -148,6 +148,42 @@ mbedtls_ecp_group_id mbedtls_ecc_group_of_psa(psa_ecc_family_t curve,
                                               int bits_is_sloppy);
 #endif /* PSA_WANT_KEY_TYPE_ECC_PUBLIC_KEY */
 
+#if defined(MBEDTLS_ASN1_WRITE_C)
```

Comment:
Yes, these are required as long as we rely on ASN1 functions. Of course this can be dropped in the parallel implementation I will start without ASN1 functions.